scripts/insert_into_appointments.sql

```diff
@@ -1 +1,6 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(time, date_of_appointment, comment, doctor_id, patient_id)
+VALUES ('10am', 'August 12, 2016', '', 4, 8),
+       ('1pm', 'August 17, 2016', '', 6, 5),
+       ('10am', 'August 12, 2016', '', 6, 8),
+       ('1pm', 'August 17, 2016', '', 4, 7)
```

Comment:
Hard-coding values like this will work, so I won't take points off, but it probably would have been a better idea to look up the specific patients and doctors from their respective tables as part of the query.